ui/src/components/UrlSummaryCard.svelte

```diff
@@ -4,11 +4,19 @@
   import { Navigate, navigateTo } from "svelte-router-spa";
   export let value = {};
   export let url;
+
+  let truncUrl = url;
+  if (url.length > 50) {
+    truncUrl = url.slice(0, 50) + "....";
```

Comment:
you don't have to do this way, you can use css for this https://tailwindcss.com/docs/word-break